extensions/amp-facebook-comments/validator-amp-facebook-comments.protoascii

```diff
@@ -28,11 +30,8 @@ tags: {  # <amp-facebook-comments>
   html_format: AMP
   tag_name: "AMP-FACEBOOK-COMMENTS"
   requires_extension: "amp-facebook-comments"
-  # data-* is generally allowed, but it's not generally mandatory.
-  attrs: {
-    name: "data-href"
-    mandatory: true
-  }
+  requires: "amp-facebook-comments 0.1"
+  attr_lists: "amp-facebook"
```

Comment:
I'd suggest leaving a comment for the reader on where to find this list, since it's in a different file.

Reply:
Done